$NetBSD: patch-aa,v 1.4 2022/06/06 05:11:58 dholland Exp $

- Remove bashisms.
- Fix itcl library names.
- Fix BLT library names.
- Avoid print from test(1) because ITCL_MAJOR_VERSION is never actually set.

--- configure.orig	2004-04-14 03:39:47.000000000 +0000
+++ configure
@@ -1430,7 +1430,7 @@ TK_LIB_STATIC=${TKCONF_PATH}/libtk${TK_V
 # Check for the TkStep libraries (if needed)...
 #
 
-if test "$ENABLE_TKSTEP" == "yes" ; then
+if test "$ENABLE_TKSTEP" = "yes" ; then
   TK_LIB_SPEC="-ltkstep -lXpm -ltiff"
   { echo "$as_me:$LINENO: will link using TkStep libraries" >&5
 echo "$as_me: will link using TkStep libraries" >&6;}
@@ -1446,7 +1446,7 @@ echo "$as_me:$LINENO: result: $TCL_VERSI
 echo "${ECHO_T}$TCL_VERSION" >&6
 if test "$TCL_MAJOR_VERSION" -lt 8; then
    echo
-   if test "$ENABLE_UNSUPPORTED" == "yes" ; then
+   if test "$ENABLE_UNSUPPORTED" = "yes" ; then
       { echo "$as_me:$LINENO: WARNING: you may be attempting to use TkDesk 2.x with an unsupported version of Tcl.  Good luck.  Hope it works out for you." >&5
 echo "$as_me: WARNING: you may be attempting to use TkDesk 2.x with an unsupported version of Tcl.  Good luck.  Hope it works out for you." >&2;}
    else
@@ -1462,7 +1462,7 @@ echo "$as_me:$LINENO: result: $TK_VERSIO
 echo "${ECHO_T}$TK_VERSION" >&6
 if test "$TK_MAJOR_VERSION" -lt 8; then
    echo
-   if test "$ENABLE_UNSUPPORTED" == "yes" ; then
+   if test "$ENABLE_UNSUPPORTED" = "yes" ; then
       { echo "$as_me:$LINENO: WARNING: you may be attempting to use TkDesk 2.x with an unsupported version of Tk.  Good luck.  Hope it works out for you." >&5
 echo "$as_me: WARNING: you may be attempting to use TkDesk 2.x with an unsupported version of Tk.  Good luck.  Hope it works out for you." >&2;}
    else
@@ -1503,9 +1503,9 @@ echo "${ECHO_T}$ITCLCONF_PATH/$ITCLCONF_
 echo $ECHO_N "checking version of Itcl... $ECHO_C" >&6
    echo "$as_me:$LINENO: result: $ITCL_VERSION" >&5
 echo "${ECHO_T}$ITCL_VERSION" >&6
-   if test "$ITCL_MAJOR_VERSION" -ne 3; then
+   if test "${ITCL_MAJOR_VERSION:-3}" -ne 3; then
       echo
-      if test "$ENABLE_UNSUPPORTED" == "yes" ; then
+      if test "$ENABLE_UNSUPPORTED" = "yes" ; then
 	 { echo "$as_me:$LINENO: WARNING: you may be attempting to use TkDesk 2.x with an unsupported version of Itcl.  Good luck.  Hope it works out for you." >&5
 echo "$as_me: WARNING: you may be attempting to use TkDesk 2.x with an unsupported version of Itcl.  Good luck.  Hope it works out for you." >&2;}
       else
@@ -1548,10 +1548,10 @@ else
      ITCL_LIB_SPEC="-L${TCLCONF_PATH} -litcl3.1"
      ITCL_LIB_STATIC=${TCLCONF_PATH}/libitcl3.1.a
   fi
-  if test -f "${TCLCONF_PATH}/libitcl3.2.so" -o -f "${TCLCONF_PATH}/libitcl3.2.a"
+  if test -f "${TCLCONF_PATH}/libitcl32.so" -o -f "${TCLCONF_PATH}/libitcl32.a"
   then
-     ITCL_LIB_SPEC="-L${TCLCONF_PATH} -litcl3.2"
-     ITCL_LIB_STATIC=${TCLCONF_PATH}/libitcl3.2.a
+     ITCL_LIB_SPEC="-L${TCLCONF_PATH} -litcl32"
+     ITCL_LIB_STATIC=${TCLCONF_PATH}/libitcl32.a
   fi
 fi
 
@@ -1567,17 +1567,17 @@ if test -z "$BLT_LIB_PATH" ; then
    echo "$as_me:$LINENO: result: ./blt" >&5
 echo "${ECHO_T}./blt" >&6
 else
-   BLT_LIB_SPEC="-L$BLT_LIB_PATH -lBLT"
-   BLT_LIB_STATIC=$BLT_LIB_PATH/libBLT.a
+   BLT_LIB_SPEC="-L$BLT_LIB_PATH -lBLT24"
+   BLT_LIB_STATIC=$BLT_LIB_PATH/libBLT24.a
    NEED_BLT_LIB=
-   if test   ! -f $BLT_LIB_PATH/libBLT.a \
-          -a ! -f $BLT_LIB_PATH/libBLT.so
+   if test   ! -f $BLT_LIB_PATH/libBLT24.a \
+          -a ! -f $BLT_LIB_PATH/libBLT24.so
    then
-      if test   ! -f $BLT_LIB_PATH/lib/libBLT.a \
-	     -a ! -f $BLT_LIB_PATH/lib/libBLT.so
+      if test   ! -f $BLT_LIB_PATH/lib/libBLT24.a \
+	     -a ! -f $BLT_LIB_PATH/lib/libBLT24.so
       then
 	 echo
-	 { { echo "$as_me:$LINENO: error: could not find libBLT.a or libBLT.so in $BLT_LIB_PATH or in $BLT_LIB_PATH/lib." >&5
+	 { { echo "$as_me:$LINENO: error: could not find libBLT24.a or libBLT24.so in $BLT_LIB_PATH or in $BLT_LIB_PATH/lib." >&5
 echo "$as_me: error: could not find libBLT.a or libBLT.so in $BLT_LIB_PATH or in $BLT_LIB_PATH/lib." >&2;}
    { (exit 1); exit 1; }; }
       else
